pybv/io.py

```diff
@@ -74,9 +74,11 @@ def write_brainvision(*, data, sfreq, ch_names, fname_base, folder_out,
         resolution, and writing data in µV with 0.1 resolution will thus
         guarantee accurate writing vor all values >= 1e-7 µV
         (``1e-6 * 0.1``).
-    unit : str
+    unit : str | list of strings, shape (n_channels,)
```

Comment:
```suggestion
    unit : str | list of strings
```
Lists don't have a shape, only a length